config/config.go

```diff
@@ -462,15 +462,15 @@ func (cfg Config) ProducerPrivateKey() keypair.PrivateKey {
 
 // ValidateChain validates the chain configure
 func ValidateChain(cfg Config) error {
-	if cfg.Chain.NumCandidates <= 0 {
+	if cfg.Chain.NumCandidates == 0 {
```

Comment:
I think the previous `<=` is fine. It just makes sure even for some bad reason uint change to type int we are still safe